src/Dependencies/Collections/SegmentedDictionary`2.cs

```diff
@@ -152,7 +153,7 @@ private void AddRange(IEnumerable<KeyValuePair<TKey, TValue>> enumerable)
             // back-compat with subclasses that may have overridden the enumerator behavior.
             if (enumerable.GetType() == typeof(SegmentedDictionary<TKey, TValue>))
             {
-                SegmentedDictionary<TKey, TValue> source = (SegmentedDictionary<TKey, TValue>)enumerable;
+                var source = (SegmentedDictionary<TKey, TValue>)enumerable;
```

Comment:
This commit (https://github.com/dotnet/roslyn/pull/72687/commits/1ecc645aa2d1424019c19339151d7b2a25b211dd) says "Apply changes for build in Roslyn" but it is not clear to me why changing explicitly typed variable to `var` would be necessary for a successful build.

Reply:
➡️ This line used `var` in Roslyn main. The switch in the commit you reference reduced the size of the final diff that will apply to code in main when this PR merges.